templates/objective-c/README.md

```diff
@@ -0,0 +1,74 @@
+# PROJECT
+
+[![CI Status](http://img.shields.io/travis/fulldecent/PROJECT.svg?style=flat)](https://travis-ci.org/fulldecent/PROJECT)
+[![Version](https://img.shields.io/cocoapods/v/PROJECT.svg?style=flat)](http://cocoadocs.org/docsets/PROJECT)
+[![License](https://img.shields.io/cocoapods/l/PROJECT.svg?style=flat)](http://cocoadocs.org/docsets/PROJECT)
+[![Platform](https://img.shields.io/cocoapods/p/PROJECT.svg?style=flat)](http://cocoadocs.org/docsets/PROJECT)
```

Comment:
would recommend these URLs go to CocoaPods.org - probably a better URL for this stuff: `http://cocoapods.org/pods/PROJECT`